hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -760,8 +767,26 @@ public ScmInfo getScmInfo() throws IOException {
         .setScmId(resp.getScmId())
         .setRatisPeerRoles(resp.getPeerRolesList());
 
-    return builder.build();
+    // By default, we assume that SCM Ratis is not enabled.
```

Comment:
We have considered whether the `ScmRatisEnabled` field exists in the proto. If the field is not present, we determine whether Ratis is enabled based on whether `peerRolesList` includes `leader` or `follower`.

Reply:
@ChenSammi Kindly ping, Can you review this PR again? Thank you very much!